src/test/fuzz/txdownloadman.cpp

```diff
@@ -373,9 +376,12 @@ FUZZ_TARGET(txdownloadman_impl, .init = initialize)
                 if (!reject_contains_wtxid) Assert(todo.m_unique_parents.size() <= rand_tx->vin.size());
             },
             [&] {
-                GenTxid gtxid = fuzzed_data_provider.ConsumeBool() ?
-                                GenTxid::Txid(rand_tx->GetHash()) :
-                                GenTxid::Wtxid(rand_tx->GetWitnessHash());
+                GenTxid gtxid;
+                if (fuzzed_data_provider.ConsumeBool()) {
+                    rand_tx->GetHash();
+                } else {
+                    rand_tx->GetWitnessHash();
+                }
```

Comment:
The return values for `GetHash` and `GetWitnessHash` need to be assigned to gtxid